e2etest/bgp_test.go

```diff
@@ -73,15 +75,36 @@ var _ = ginkgo.Describe("BGP", func() {
 		hostport := net.JoinHostPort(ingressIP, port)
 		address := fmt.Sprintf("http://%s/", hostport)
 
-		useDocker := true
+		containerName := "frr"
 		if skipDockerCmd {
 			ginkgo.By("checking connectivity to its external VIP")
-			useDocker = false
+			containerName = ""
 		} else {
 			ginkgo.By("checking connectivity to its external VIP with docker")
 		}
-		err = wgetRetry(useDocker, address)
+
+		err = wgetRetry(containerName, address)
+		framework.ExpectNoError(err)
+
+		allNodes, err := cs.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
```

Comment:
I guess the e2e test is run with a timeout and aborts if this stalls? Or do we want to specify a timeout in the context here?

Reply:
Yes, moreover it's kind local to the test runner. I'd avoid bloating the tests with timed contexts.